jmix-security/security-flowui/src/main/java/io/jmix/securityflowui/authentication/LoginViewSupport.java

```diff
@@ -389,6 +391,18 @@ protected TimeZone getDeviceTimeZone() {
         return deviceTimeZoneProvider.getDeviceTimeZone();
     }
 
+    protected RouteConfiguration getRouteConfiguration() {
+        return viewRegistry.getRouteConfiguration();
+    }
+
+    @Nullable
+    protected String removeLeadingSlash(@Nullable String path) {
```

Comment:
Let's name it more generic, e.g. `normalizePath` or `cleanPath`

Reply:
Fixed, replaced by `LocationUtil#ensureRelativeNonNull()`